composer.json

```diff
@@ -11,14 +11,14 @@
 	"license": "MIT",
 	"require": {
 		"composer/installers": ">=1.0.1",
-		"sentry/sentry": "0.12.1"
+		"sentry/sentry": "0.20.0"
```

Comment:
it moves me crazy, why cannot I see real difference between the branches?   it is already "0.20.0" in our master branch

Reply:
hmmm .. weird, either way, we're on 3.8.x version already so we can't sync with the wikimedia repo